src/python/pants/backend/python/tasks2/setup_py.py

```diff
@@ -278,6 +278,8 @@ class SetupPy(Task):
 
   SOURCE_ROOT = b'src'
 
+  PRODUCT_TYPE = 'python_dists'
```

Comment:
Better to call this something like PYTHON_DISTS_PRODUCT, in case this task is ever modified to produce more than one product in the future.